.travis.yml

```diff
@@ -1,17 +1,25 @@
 language: python
+
+python:
+- "2.7"
+- "3.5"
+env:
+- TOX_FLAVOR=default
+- TOX_FLAVOR=fido
+
 matrix:
   include:
-  - python: "3.5"
-    env: TOX_ENV=py35
-env:
-- TOX_ENV=py27
-- TOX_ENV=cover
-- TOX_ENV=docs
-- TOX_ENV=flake8
+  - python: "2.7"
+    env: TOX_ENV=cover
+  - python: "2.7"
+    env: TOX_ENV=docs
+  - python: "2.7"
+    env: TOX_ENV=flake8
```

Comment:
Instead of being cute, let's be explicit:

```yaml
matrix:
  include:
  - env: TOXENV=py27-default
  - env: TOXENV=py27-fido
  - env: TOXENV=py36-default
    python: python3.6
  - env: TOXENV=py36-fido
    python: python3.6
  - env: TOXENV=cover
  - env: TOXENV=docs
  - env: TOXENV=flake8
```

This is _much_ clearer what's going on :)